.travis.yml

```diff
@@ -6,7 +6,7 @@
 # - sudo/dist/group are set so as to get Blue Box VMs, necessary for [loopback]
 #   IPv6 support
 
-sudo: false
+sudo: required
```

Comment:
Regarding to this https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments
This will change ci environment from "CONTAINER-BASED" to be "STANDARD" which has no cache for public and longer boot time.
If you are going to follow bitcoin to use "TRUSTY BETA" environment, more changes would need to be done.